### ISSUE TITLE
Search: Not all overlay elements focusable in Safari

### ISSUE THREAD

**Issue by annacmc** [Automattic/jetpack#25261]
### Impacted plugin

Search

### Steps to Reproduce

* In Safari, Go to a site with Instant Search enabled and open the search modal.
* Use the keyboard to navigate through the page with Tab.

### A clear and concise description of what you expected to happen.

I expect to be able to tab through the items, for example:

`input > clear > search > sort relevance > sort newest > sort oldest > close modal > results > filters`

### What actually happened

In Safari, I am only able to tab through the following sequence:

`input > sort relevance > browser address bar` 

Using the tab key, I am unable to access the other elements (such as results, filters etc)

### Reproducibility

Consistent

**Comment by annacmc** [comment]
Closing this up as it turns out it's a Safari setting which can be resolved in `Preferences → Advanced → Accessibility `
